runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiSearchChatboxTextInput.java

```diff
@@ -297,7 +296,7 @@ public void keyPressed(KeyEvent ev)
 
 	private void search(String search)
 	{
-		LinkBrowser.browse(WikiPlugin.WIKI_BASE + "?search=" + URLEncoder.encode(search) + "&" + WikiPlugin.UTM_PARAMS);
+		LinkBrowser.browse(HttpUrl.parse(WikiPlugin.WIKI_BASE + "?search=" + search + "&" + WikiPlugin.UTM_PARAMS).toString());
```

Comment:
🤦‍♂️

Reply:
```
		new HttpUrl.Builder()
			.addQueryParameter("blah")
			.addQueryParameter("blaah")
			.build()
			.toString()
```

Reply:
`HttpUrl.parse` uses the `HttpUrl.Builder` under the hood, but if you really want it to be used directly, I can do that too.

Reply:
Maybe you should do some reading on URL encoding and why it is required before attempting this change.

Reply:
I already knew what URL encoding is, but upon review, I see why using `HttpUrl.parse` in that way would not work properly - my mistake. Commit 5b3708d now uses `HttpUrl.Builder`.